### PR TITLE
refactor(movies): switched to async/await for movie controller

### PR DESCRIPTION
**Pull request by robintsunny**
what: refactored the movie controller from promises to async/await
why: a/a is better

### CODE REVIEW

File: test/plugins/features/movies/index.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const Movies = require('../../../../lib/server');
+
+describe('movies integration', () => {
+
+  describe('create', () => {
+
+    it('creates a movie', () => {
+      return Movies.inject({
```
Comment (sjl2) [inline test/plugins/features/movies/index.test.js:10]: If this test is being added, can it be made a/a?

Comment (robintsunny) [reply test/plugins/features/movies/index.test.js:10]: i think i added that in earlier but forgot to give it its own commit, so it got lumped in with this commit. I plan on making the changes for test files after the controller gets approval

Comment (sjl2) [reply test/plugins/features/movies/index.test.js:10]: Ok, but the functionality is going into this commit. 

I guess it's fine for now.

File: test/plugins/features/movies/index.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        url: '/movies',
+        method: 'POST',
+        payload: { title: 'Volver' }
+      })
+        .then((response) => {
```
Comment (sjl2) [inline test/plugins/features/movies/index.test.js:15]: nit: can you align this promise chain?

**Review by sjl2 — APPROVED** [review]
LGTM

File: test/plugins/features/movies/index.test.js (hunk 2 above)
Comment (robintsunny) [reply test/plugins/features/movies/index.test.js:15]: got it

**Review by ajpawlicki — APPROVED** [review]

**Pull request merged**